third_party/nvfuser/csrc/python_frontend/fusion_record.h

```diff
@@ -443,14 +443,14 @@ struct SqueezeOpRecord : RecordFunctor {
       std::vector<State> _args,
       std::vector<State> _outputs,
       std::vector<int64_t>& original_shape,
-      int64_t dim)
+      std::vector<int64_t>& dims)
```

Comment:
So with this change, we can no longer accept a single number for `dim` any more? This is getting ugly....
I'm advocating to push our user facing API from pybind to real python scripts, so these can be easily handled in python, other than the cumbersome pybind overload.
https://github.com/csarofeen/pytorch/pull/2430#issuecomment-1423037712

Reply:
Just to make it clear, not suggesting that we should do it in this PR. We can merge it as-is